src/web/wms/src/main/java/org/geoserver/wms/web/data/ExternalGraphicPanel.java

```diff
@@ -64,6 +65,8 @@
 
     private Model<String> showhideStyleModel = new Model<String>("");
 
+    private static String externalURIPattern = "^(http|https|file)://.*$";
```

Comment:
Not looking for a file here, this is not used to access any file on the file system - instead only used to access a resource in the styles folder (ideally via Resource API.)

Reply:
that functionality still holds and untouched, the unit tests assert that